src/containers/subjects/AnalysisResultTable/index.tsx

```diff
@@ -486,6 +496,23 @@ function groupResultsData({
     (r) => r.path.includes('tso_ctdna') && r.path.endsWith('tsv')
   );
 
+  // CTTSOV2 results
+  const cttsov2Bams = results_cttsov2.filter(
+    (r) => r.key.includes('cttsov2') && r.key.endsWith('bam')
+  );
+  const cttsov2Vcfs = results_cttsov2.filter(
+    (r) => r.key.includes('cttsov2') && (r.key.endsWith('vcf') || r.key.endsWith('vcf.gz'))
```

Comment:
As discussed. Let filter `.vcf` to exclude `.gvcf` files.

Putting a dot should do it.
```
(r) => r.key.includes('cttsov2') && (r.key.endsWith('.vcf') || r.key.endsWith('.vcf.gz'))
```

Reply:
sure, added